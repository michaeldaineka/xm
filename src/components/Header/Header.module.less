@import 'styles/app';

.header {
  position: fixed;
  top: 0;
  background-color: #000;
  height: 85px;
  left: 0;
  right: 0;
  display: flex;
  align-items: center;
  padding: 0 24px;
  z-index: 999;
  .small {
    color: @white;
    align-self: end;
    padding-left: 5px;
  }
}
