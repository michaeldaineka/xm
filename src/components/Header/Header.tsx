import React from 'react';
import s from './Header.module.less';
import { Row, Col } from 'antd';
import logo from 'images/logo.svg';

const Header = (): JSX.Element => {
  return (
    <header className={s.header}>
      <Row>
        <Col sm={24} className={'d-flex'}>
          <img src={logo} alt="XM Logo" width="147.88px" height="39" data-test-id={"logo"} />
          <small className={s.small}>task</small>
        </Col>
      </Row>
    </header>
  );
};

export default Header;
