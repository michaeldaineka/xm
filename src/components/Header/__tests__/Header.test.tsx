import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

test('Alert component renders', () => {
    const component = renderer.create(
        <Header />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
