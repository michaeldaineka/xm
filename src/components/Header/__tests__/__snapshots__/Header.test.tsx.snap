// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert component renders 1`] = `
<header>
  <div
    className="ant-row"
    style={Object {}}
  >
    <div
      className="ant-col d-flex ant-col-sm-24"
      style={Object {}}
    >
      <img
        alt="XM Logo"
        height="39"
        src="logo.svg"
        width="147.88px"
      />
      <small
        className={[Function]}
      >
        task
      </small>
    </div>
  </div>
</header>
`;

exports[`Alert component renders 2`] = `
<header>
  <div
    className="ant-row"
    style={Object {}}
  >
    <div
      className="ant-col d-flex ant-col-sm-24"
      style={Object {}}
    >
      <img
        alt="XM Logo"
        height="39"
        src="logo.svg"
        width="147.88px"
      />
      <small
        className={[Function]}
      >
        task
      </small>
    </div>
  </div>
</header>
`;

exports[`Alert component renders 3`] = `
<header>
  <div
    className="ant-row"
    style={Object {}}
  >
    <div
      className="ant-col d-flex ant-col-sm-24"
      style={Object {}}
    >
      <img
        alt="XM Logo"
        height="39"
        src="logo.svg"
        width="147.88px"
      />
      <small
        className={[Function]}
      >
        task
      </small>
    </div>
  </div>
</header>
`;
